Public/Src/Cache/ContentStore/Library/Logging/FileLog.cs

```diff
@@ -179,7 +179,7 @@ public virtual void Write(DateTime dateTime, int threadId, Severity severity, st
                     message
                 );
 
-            WriteLineInternal(severity, line);
+            WriteLine(severity, severity.ToString(), line);

```

Comment:
just FYI on full framework this does extra boxing. But since you are diong advanced string formatting already I assume this is not too perf-critical

Reply:
Just realized that were already capturing the severityName in a non-boxing way. Nice catch!

---
In reply to: [313590070](https://github.com/microsoft/BuildXL/pull/729#discussion_r313590070) [](ancestors = 313590070)